examples/pylab_examples/bar_stacked.py

```diff
@@ -1,3 +1,13 @@
+"""
+=================
+Stacked Bar Graph
+=================
+
+This is an example of creating a stacked bar plot with error bars using
+`plt.bar`. Note the parameters `yerr` used for error bars, and `bottom`
```

Comment:
yerr and bottom should use `*foo*`, not backticks (they are not references to linkable elements).